cypress/integration/precise-rounding.js

```diff
@@ -0,0 +1,395 @@
+const csvtojson = require('csvtojson');
+const dayjs = require('dayjs');
+const isBetween = require('dayjs/plugin/isBetween');
+
+dayjs.extend(isBetween);
+
+const testCaseOne = {
```

Comment:
**suggestion (non-blocking):** It might be good to stash all these in a fixture file, see [these examples](https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/fundamentals__fixtures) and [these docs](https://docs.cypress.io/api/commands/fixture)